client/app/vps/translations/Messages_pl_PL.xml

```diff
@@ -43,6 +43,11 @@
    <translation id="vps_monitoring_cpu_sys" qtlid="83733">System</translation>
    <translation id="vps_monitoring_cpu_user" qtlid="7450">Użytkownik</translation>
 
+   <translation id="vps_dashboard_upgrade_1">A major update of the infrastructure VPS 2016 & 2018 in area os-sbg1-006, os-sbg1-010, os-sbg1-011, os-sbg1-012 and os-sbg1-013 is scheduled on Wednesday 11th July from 9 AM UTC+2 to 1 PM UTC+2.</translation>
```

Comment:
You should avoid using `&` character, it will be problematic for the translator :)

Please use `&#amp;`